packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -6,6 +6,7 @@
       <va-collapse
         v-for="(route, key) in navigationRoutes"
         :key="key"
+        :focusable="false"
```

Comment:
What's the logic here?

Reply:
Removed.

Reply:
Actually, no. The prop itself is removed, but forgot this usage. TODO

Reply:
removed in ec0ed78 (will push it shortly)